tests/basics/memoryview1.py

```diff
@@ -36,3 +36,67 @@
 print(list(m[1:-1]))
 m[2] = 6
 print(a)
+
+# test slice assignment between memoryviews
+b1 = bytearray(b'1234')
+b2 = bytearray(b'5678')
+b3 = bytearray(b'5678')
+m1 = memoryview(b1)
+m2 = memoryview(b2)
+m3 = memoryview(b3)
+try:
+    m2[1:3] = m1[0:2]
+except Exception as e:
+    print(e)
+else:
+    print(b2)
+
+try:
+    b3[1:3] = m1[0:2]
+except Exception as e:
+    print(e)
+else:
+    print(b3)
+
+try:
+    m1[2:4] = b3[1:3]
+except Exception as e:
+    print(e)
+else:
+    print(b1)
+
+try:
+    m2[1:3] = b1[0:4]
+except ValueError:
+    print("ValueError")
+except Exception as e:
+    print(e)
```

Comment:
For here, if you expect the assignment to fail with ValueError then you just need the `except ValueError` block, not also the general `except Exception as e` block.